test/perf/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+const webpack = require('webpack');
+const webpackBaseConfig = require('../../webpackBaseConfig');
+
+module.exports = {
+  ...webpackBaseConfig,
+  entry: path.resolve(__dirname, 'index.js'),
+  mode: 'development',
+  output: {
+    path: path.resolve(__dirname, '../../tmp'),
+    filename: 'tests.js',
+  },
+  // Avoid bundling the whole @material-ui/icons package. x2 the bundling speed.
+  plugins: [new webpack.IgnorePlugin(/material-icons\/SearchIcons\.js/)],
```

Comment:
I don't think that it's necessary, we don't import any file from the documentation

```suggestion
```